pkg/metrics/otel.go

```diff
@@ -121,6 +121,39 @@ processors:
           - cluster_scoped_resource_count
           - resource_ns_count
           - api_duration_seconds
+  # Aggregate some metrics sent to Cloud Monitoring to remove high-cardinality labels (e.g. "commit")
+  metricstransform/cloudmonitoring:
+    transforms:
+      - include: last_sync_timestamp
+        action: update
+        operations:
+          - action: aggregate_labels
+            label_set:
+              - configsync.sync.kind
+              - configsync.sync.name
+              - configsync.sync.namespace
+              - status
+            aggregation_type: max
+      - include: declared_resources
+        action: update
+        new_name: current_declared_resources
+        operations:
+          - action: aggregate_labels
+            label_set:
+              - configsync.sync.kind
+              - configsync.sync.name
+              - configsync.sync.namespace
+            aggregation_type: max
```

Comment:
sum is incorrect for timestamps and durations. We have to use either min, max, or mean.

Since most of these metrics are used to discover outliers, I suspect people want to know the max, more than the avg or min.

Reply:
For example, i could imagine customers making alerts based on syncs that take too long or have too many objects. But no one would make an alert for them being too fast.

Reply:
Sounds good.